ios/Podfile

```diff
@@ -71,7 +71,7 @@ post_install do |installer|
           'PERMISSION_LOCATION=0',
 
           ## dart: PermissionGroup.notification
-          'PERMISSION_NOTIFICATIONS=0',
+          'PERMISSION_NOTIFICATIONS=1',
```

Comment:
why we need change it?

Reply:
Notification permission on iOS will be permanentlyDenied if we don't change it. Also, see the [docs](https://pub.dev/packages/permission_handler)